python2/raygun4py/http_utilities.py

```diff
@@ -15,12 +16,16 @@ def build_wsgi_compliant_request(request):
     if http_host is not None:
         http_host = http_host.replace(' ', '')
 
-    # TODO, better understand how to "properly" fallback to wsgi.input stream
-    http_form = request.get('form') or request.get('wsgi.input')
-    if http_form is not None:
-        http_form = dict(http_form)
-
     try:
+        http_form = getattr(request, 'form', None) or request.get('form')
+
+        # fallback to wsgi.input
+        if http_form is None and 'wsgi.input' in request:
+            # we can assume WSGI keys inside this block
+            content_length = int(request.get('CONTENT_LENGTH', 0))
+            if content_length:
+                http_form = request['wsgi.input'].read(content_length)
```

Comment:
So http_form will just contain the content length?

Reply:
@slin63 The string of the content `read(content_length)` will turn the binary stream into a string

Reply:
👍